src/database/migrations/20210507135009_create_user.ts

```diff
@@ -8,7 +8,7 @@ const createUserAccountsTable = async (knex: Knex): Promise<void> => {
     table.string("email").nullable().unique();
     table.string("password").notNullable();
     table.string("role").nullable().defaultTo('user');
-    table.string("is_authenticated").defaultTo('false');
+    table.boolean("is_authenticated").defaultTo('false');
```

Comment:
What about renaming this to `is_verified` so that it's more clear this is a verification flag

Reply:
sure, is_verified is better in this case